modules/Bio/EnsEMBL/Compara/Production/EPOanchors/TrimAnchorAlign.pm

```diff
@@ -161,6 +148,7 @@ sub _dump_fasta {
       $self->complete_early( "Not enough sequences in current anchor - omitting" );
   }
 
+  my $fasta_files;
```

Comment:
```suggestion
  my $fasta_files = '';
```

Reply:
I was hoping Perl was clever enough to do this, but I guess it is not :'(